src/app/frontend/replicasetlist/replicasetlist_controller.js

```diff
@@ -28,10 +29,24 @@ export default class ReplicaSetListController {
    */
   constructor($state, replicaSets) {
     /** @export {!Array<backendApi.ReplicaSet>} */
-    this.replicaSets = replicaSets.replicaSets;
+    this.replicaSets = replicaSets.replicaSets || [];
```

Comment:
Hmm... Are you sure that when there are no replica sets the array is null? If yes, can you modify backend to always return a non null array instead of being defensive here?